numpyro/infer/mcmc.py

```diff
@@ -351,7 +349,8 @@ def _single_chain_mcmc(self, init, args, kwargs, collect_fields):
                                     thinning=self.thinning,
                                     collection_size=collection_size,
                                     progbar_desc=partial(_get_progbar_desc_str, lower_idx, phase),
-                                    diagnostics_fn=diagnostics)
+                                    diagnostics_fn=diagnostics,
+                                    num_chains=self.num_chains)
```

Comment:
could you change this to: `num_chains=self.num_chains if self.chain_method == "parallel" else 1` (if you prefer clarity, we can change `num_chains` to `num_parallel_chains`)

Reply:
done! (in the recent commit)

Reply:
thanks!